Thunder/server/stream_routes.py

```
@@ -123,11 +123,11 @@ async def media_preview(request: web.Request):
         return web.Response(text=rendered_page, content_type='text/html')
         
     except (InvalidHash, FileNotFound) as e:
-        logger.debug(f"Client error in preview: {type(e).__name__}")
+        logger.debug(f"Client error in preview: {type(e).__name__}", exc_info=True)
         raise web.HTTPNotFound(text="Resource not found")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)